app-android/src/main/java/com/ustadmobile/port/android/view/XapiPackageContentFragment.kt

```diff
@@ -0,0 +1,106 @@
+package com.ustadmobile.port.android.view
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.webkit.WebChromeClient
+import android.webkit.WebSettings
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import com.toughra.ustadmobile.databinding.FragmentXapiPackageContentBinding
+import com.ustadmobile.core.controller.XapiPackageContentPresenter
+import com.ustadmobile.core.util.ext.toNullableStringMap
+import com.ustadmobile.core.util.ext.toStringMap
+import com.ustadmobile.core.view.MountedContainerHandler
+import com.ustadmobile.core.view.XapiPackageContentView
+import com.ustadmobile.sharedse.network.NetworkManagerBle
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+
+class XapiPackageContentFragment : UstadBaseFragment(), XapiPackageContentView {
+
+    var networkManagerProvider: BleNetworkManagerProvider? = null
+
+    override fun onAttach(context: Context) {
```

Comment:
Because networkManagerProvider is only accessible for testing purposes it would be good to use the internal modifier. Internal allows access only within the same module and the test code for that module.